### ISSUE TITLE
session start error

### ISSUE THREAD

**Issue by danielpm307** [SafeExamBrowser/seb-win-refactoring#311]
**IMPORTANT**
Please _always_ consult the documentation first before creating a bug report!
https://safeexambrowser.org/windows/win_usermanual_en.html

**Describe the Bug**
The safe browser occurs an error while lunching.

**Steps to Reproduce**
1. open 'safe browser.'
2. launching ...
3. See an error

**Expected Behavior**
open a safe browser for entering the exam website. 
other students have no problem with that .its just about conflict with my system.
**Screenshots**
![image](https://user-images.githubusercontent.com/1260701/147637488-0c69f938-5a48-46c8-9134-aaadaa7b05ec.png)

**Version Information**
 - OS: win 11 22523
 - SEB-Version 3.3.1

**Additional Context**
log file:
[1400-10-08_10h55m19s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7787716/1400-10-08_10h55m19s_Client.log)
[1400-10-08_10h55m19s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7787717/1400-10-08_10h55m19s_Runtime.log)

**Comment by danielpm307** [comment]
for additional info : 
- I uninstall and updated SEB but no change. 
- other students have no problem with that. (this error depends on my system)
- My RDP Patch has enabled multi-user to connect without logout problems. (custom RDP)

**Comment by danielpm307** [comment]
I Fixed it; there was a problem with the input problem of windows 11 insider. I deleted a language, but a few registry keys were not deleted from the windows registry in language deleting. 
I delete It manually. It fixed. 

Good luck.

**Comment by dbuechel** [comment]
Great, glad to know that you were able to solve the issue. To help us in case this issue happens to other users and/or potentially improve our code, would you mind elaborating a bit more about the exact reason of failure (you mentioned an input language problem in Windows 11 Insider) as well as how exactly you solved it (which language caused the failure and which registry keys you needed to delete)?

The application log indeed shows an issue with an input language, it however doesn't reveal much more:
```
1400-10-08 10:55:20.956 [01] - ERROR: Caught unexpected exception while performing operation 'LazyInitializationOperation'!

   Exception Message: Culture is not supported.
Parameter name: culture
4096 (0x1000) is an invalid culture identifier.
   Exception Type: System.Globalization.CultureNotFoundException

   at System.Globalization.CultureInfo.InitializeFromCultureId(Int32 culture, Boolean useUserOverride)
   at System.Windows.Input.InputProcessorProfiles.get_InputLanguageList()
   at SafeExamBrowser.SystemComponents.Keyboard.Keyboard.Initialize() in C:\Users\appveyor\projects\seb-win-refactoring\SafeExamBrowser.SystemComponents\Keyboard\Keyboard.cs:line 54
   at SafeExamBrowser.Client.Operations.ShellOperation.InitializeSystemComponents() in C:\Users\appveyor\projects\seb-win-refactoring\SafeExamBrowser.Client\Operations\ShellOperation.cs:line 207
   at SafeExamBrowser.Client.Operations.ShellOperation.Perform() in C:\Users\appveyor\projects\seb-win-refactoring\SafeExamBrowser.Client\Operations\ShellOperation.cs:line 81
   at SafeExamBrowser.Core.OperationModel.OperationSequence.Perform() in C:\Users\appveyor\projects\seb-win-refactoring\SafeExamBrowser.Core\OperationModel\OperationSequence.cs:line 108
```

**Comment by danielpm307** [comment]
in this case i installed Germany language (both)
![image](https://user-images.githubusercontent.com/1260701/147962855-51067f2d-3341-418b-8fb5-d7f02a1f4a4a.png)

And for a lot of input options, I deleted English(Germany). 
After I deleted it, the language wasn't deleted, and I had input and some text about that language. 
I searched on the internet, and I deleted additional input keys of it in the registry. 

- HKEY_USERS\.DEFAULT\Keyboard Layout\Preload
- HKEY_CURRENT_USER\Keyboard Layout\Preload
- HKEY_USERS\.DEFAULT\Control Panel\International\User Profile
- HKEY_USERS\.DEFAULT\Control Panel\International\User Profile System Backup

I think it's an insider build problem or a rare case.
